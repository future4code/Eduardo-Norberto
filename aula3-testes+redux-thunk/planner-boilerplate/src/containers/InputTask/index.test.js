import React from "react";
import renderer from "react-test-renderer";
import Grid from '@material-ui/core/Grid';
import { mapDispatchToProps } from "./index"


describe("Testar o InputTask", ()=>{
    test("Snapshot", ()=>{
        const testSnapshot = renderer.create(<Grid />).toJSON();
        expect(testSnapshot).toMatchSnapshot();
    })  
})

describe('mapDispatchToProps', () => {
    it('Testa se o mapDispatchToProps foi chamado', () => {
        const dispatch = jest.fn();
        mapDispatchToProps(dispatch).createTask();
        expect(dispatch).toHaveBeenCalledTimes(1);
    });
});

