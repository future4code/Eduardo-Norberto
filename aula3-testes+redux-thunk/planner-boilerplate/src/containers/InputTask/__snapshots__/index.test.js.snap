// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar o InputTask Snapshot 1`] = `
<div
  className="MuiGrid-root"
/>
`;
